### ISSUE TITLE
Design Intent - Design Feedback - EDM and VEBT team, GI Bill Comparison Tool homepage

### ISSUE THREAD

**Issue by allison0034** [department-of-veterans-affairs/va.gov-team#99543]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [x] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [x] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [x] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [x] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Listen to the recording for any additional feedback from Ryan Thurlwell and Matt Dingee.
- Ryan Thurlwell: It may be time to rename this page to GI Bill Comparison Tools or Schools because we decided to break this into 3 separate things. The job of this page is to get you to the tools.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] Must: We like to limit the max width character count to 75 characters for legibility reasons. Currently you paragraph is full width and has more than 75 characters. Most designs on VA.gov use the usa-width-two-thirds and medium-8 classes to limit the width on desktop. Look at the [grid in the design system](https://design.va.gov/foundation/layout/flexbox-grid) and use 9 columns instead of 12.
- [x] Should: Matt Dingee There is not an official rule to not use headers as links but the design system team probably needs to make one. Explore other options. My Healthy Vet landing page might be a good place to look for similar things.

## Governance team actions
- [ ] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [ ] Add the touchpoint labels
- [ ] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone

**Comment by arianaadili** [comment]
@humancompanion-usds It is unclear why changing the links as headers has been marked as a "must" in the feedback, particularly since it was stated that there is no current rule against their use. The design meets our client’s requirements and has received their approval. Additionally, I have seen links used as headers across VA.gov, including on the homepage (screenshot attached). If this approach is acceptable on the homepage, I do not understand why it is flagged here. Moving forward, we will retain these elements in our design while incorporating the other feedback provided.

<img width="797" alt="Image" src="https://github.com/user-attachments/assets/21fd965d-974e-448a-91b6-b33b70cbf027" />

**Comment by allison0034** [comment]
Hi @arianaadili, as Matt said in the meeting, this is not in the design system but should be as it is an accessibility issue. I took notes for Matt and added this as a "must" item. I will defer to Matt for the final decision.

**Comment by humancompanion-usds** [comment]
@arianaadili - For now this will be a should rather than a must. When I add this as a rule to the Design System I'll be sure to circle back around to your team to ensure you fix it then (and the home page and all of the other places where we are incorrectly combining headers and links). We gave an example that would be simple to follow.

**Comment by ameliahackingVA** [comment]
I'm curious to learn about any accessibility testing that has shown results indicating issues with using links as headers. Do you have any resources or findings you could share? It's a very common practice across the web, so I'm just wondering what is problematic about it.

**Comment by humancompanion-usds** [comment]
@briandeconinck - Can you give Amelia some intel on why we try to avoid links in headers? If you and @rsmithadhoc want to write up some guidance around this I can drop it into the Design System.

**Comment by arianaadili** [comment]
Hey @allison0034 Regarding your comment about switching to the 9-column grid, our dev lead @GovCIOLiz implemented this change, assuming that you're suggesting the container should take up 9 out of 12 columns (which would be 75% width). However, as you can see in the attached screenshot, the resulting UI doesn't appear visually balanced. Could you confirm if this aligns with your expectations?

<img width="1131" alt="Image" src="https://github.com/user-attachments/assets/286d48dd-b837-436a-9666-94a95088716b" />

**Comment by allison0034** [comment]
Hi @arianaadili , I apologize, I dont work with the code typically, but there should be design tokens and VA grids you can use. This would only apply to full width content. Looking in prod, the CSS seems to be usa-width-two-thirds medium-8 columns. If I look at your screen shot, that would be applied to your "Discover how..." content. The 3 columns below would not need that. Does that make sense?

**Comment by arianaadili** [comment]
> Hi [@arianaadili](https://github.com/arianaadili) , I apologize, I dont work with the code typically, but there should be design tokens and VA grids you can use. This would only apply to full width content. Looking in prod, the CSS seems to be usa-width-two-thirds medium-8 columns. If I look at your screen shot, that would be applied to your "Discover how..." content. The 3 columns below would not need that. Does that make sense? Again,I don't work with the code, but you can check out [grid system](https://design.va.gov/foundation/layout/page-layouts) and tokens to see what should be applied where.

Hey @allison0034 if I'm understanding correctly, it sounds like you're saying the "Discover how.." text shouldn't take up the full width--is that right? I know EDM has their midpoint review tomorrow, so hopefully, we can get some additional clarity from you or others during that time. Just a heads up that this specific feedback may not be fully addressed by then, but know it's on our radar, we'll make sure it's applied by the staging review

**Comment by allison0034** [comment]
@arianaadili yes, that is correct. If you look at va.gov, we should not have any single column full width content. [Here is a good example of the max width](https://www.va.gov/health-care/apply-for-health-care-form-10-10ez/introduction).

**Comment by cassidy-beach** [comment]
**Must: We like to limit the max width character count to 75 characters for legibility reasons.** 
![Image](https://github.com/user-attachments/assets/bfe8412a-0f08-4667-88d6-9c0ba60b09fd)

**Comment by cassidy-beach** [comment]
Should: Matt Dingee There is not an official rule to not use headers as links but the design system team probably needs to make one
Created [VEBT-1368](https://jira.devops.va.gov/browse/VEBT-1368) to address this issue at a later date
